### DISCUSSION TITLE
Skim

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [elrumo/macOS_Big_Sur_icons_replacements#1200] [Icon request]
Request for a [Skim](https://skim-app.sourceforge.io) icon. I would be very appreciative

Thank you

**Comment by blaichch** [comment]
As far as I can see Skim already has a Big Sur icon. Are you just looking for an alternative icon?

**Reply by tsuliwaensis** [reply]
Yes just looking for an alternative icon

**Reply by maxyvon** [reply]
What are you looking for? What differences would you like?

**Reply by tsuliwaensis** [reply]
I didn't have anything in mind. I just like alternative icons on my apps 😃
